backend/src/ApplicationCore/Entities/Orders/Order.cs

```diff
@@ -0,0 +1,41 @@
+﻿using PartyKlinest.ApplicationCore.Entities.Orders.Opinions;
+using PartyKlinest.ApplicationCore.Entities.Users;
+using PartyKlinest.ApplicationCore.Entities.Users.Cleaners;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PartyKlinest.ApplicationCore.Entities.Orders
+{
+    public class Order
+    {
+        public long OrderId { get; set; }
+        
+        public DateTimeOffset TimeFrom { get; set; }
+        public DateTimeOffset TimeTo { get; set; }
+
+        public decimal MaxPrice { get; set; }
+        public int MinCleanerRating { get; set; }
+        
+        public MessLevel MessLevel { get; set; }
+        public OrderStatus Status { get; set; }
+
+
+        public long ClientId { get; set; }
+        public Client Client { get; set; }
+
+        public long? CleanerId { get; set; }
```

Comment:
cleanerId jest stringiem (według dokumentacji)

Reply:
Wydaje mi się, że jest tak tylko w endpointach, tutaj dane z modułu osoby sprzątającej:
![image](https://user-images.githubusercontent.com/62249621/160694349-d967567b-cf77-439b-b829-8e86a27e4e52.png)
Zastanawiam się czy opłaca wyróżnić cleanerId, które jest nadawane po kolei przez postgres przy dodaniu wiersza, oraz trzymać id z azure jako oddzielne pole, czy po prostu mieć tylko id z azure jako klucz główny.

Reply:
Myślę, że dla spójności można trzymać azurowe id

Reply:
To zmieniłem domyślne id klienta i cleanera na string gdzie będziemy wrzucać po prostu azurowe id.